### ISSUE TITLE
Edit & Update: Tearsheet

### ISSUE THREAD

**Issue by andrea-island** [carbon-design-system/ibm-products#1260]
# Name of component or pattern

Edit & Update: Tearsheet (wide)

## Brief description

Similar to the Create Tearsheet, the Edit/Update Tearsheet will be used to edit assets that have many attributes/fields and sections.

## Details

[Link to Design](https://pages.github.ibm.com/cdai-design/pal/patterns/edit#wide-tearsheet-edit)

NOTE: There are open questions with this design. Recommend ensuring all is addressed before starting.

### Links to other materials

Check out my post in #ccs-pal on Sep 14

## Design Maintainer(s)

- Shelley Dignum

## Tasks

Before starting work on this epic, please review and complete the following.

- [x] Initial review of design/existing code

### Working in Cloud and Cognitive package

- [x] Have you reviewed our
      [contribution guidelines](https://github.com/carbon-design-system/ibm-cloud-cognitive/blob/master/.github/CONTRIBUTING.md)?
- [ ] Have you noted our
      [code guidelines](https://github.com/carbon-design-system/ibm-cloud-cognitive/blob/master/docs/CODE_GUIDELINES.md)?
- [ ] Did you use the CLI to scaffold your new component?

**Comment by lee-chase** [comment]
The PAL site will need updating to reflect the merging of these two patterns. 

@Ratheeshrajan please raise an issue here https://github.ibm.com/CDAI-design/pal/issues asking for the patterns to be merged, referring to this issue and your conversation with Shelley. 

Please tag or assign the new issue to Shelley.

**Comment by Ratheeshrajan** [comment]
> The PAL site will need updating to reflect the merging of these two patterns.
> 
> @Ratheeshrajan please raise an issue here https://github.ibm.com/CDAI-design/pal/issues asking for the patterns to be merged, referring to this issue and your conversation with Shelley.
> 
> Please tag or assign the new issue to Shelley.

@lee-chase Many thanks for your comment. I'll connect with Shelley and discuss merging the Create/Edit patterns.

**Comment by elycheea** [comment]
@Ratheeshrajan Were you able to follow up with Shelley on the Create/Edit Tearsheet pattern?

We’re also looking into whether or not we need to open another ticket for Modal and CreateFullPage, but I expect they’ll fall into a similar bucket?

**Comment by elycheea** [comment]
@Ratheeshrajan Can we add examples for editing the Tearsheet in under the **Edit and update** pattern in Storybook? Don’t think we need to add entire components but examples that follow the PAL guidance using the existing components would be sufficient.

https://pages.github.ibm.com/cdai-design/pal/patterns/edit/usage/#tearsheet-edit

**Comment by davidmenendez** [comment]
working on review now. just wanted to start by saying the tests appear to be failing. probably want to start by fixing those tests.

**Comment by Solly2** [comment]
Is there an update on this at all?  We (Project Neptune) need to use the Edit and Update pattern in our GA release (June 2023)!

**Comment by jagathgj** [comment]
Fixed and merged it. Hence closing.
